rootston/cursor.c

```diff
@@ -294,16 +298,18 @@ static void roots_cursor_press_button(struct roots_cursor *cursor,
 
 void roots_cursor_handle_motion(struct roots_cursor *cursor,
 		struct wlr_event_pointer_motion *event) {
-	wlr_cursor_move(cursor->cursor, event->device,
-			event->delta_x, event->delta_y);
-	roots_cursor_update_position(cursor, event->time_msec);
+	if (!cursor->locked) {
```

Comment:
It occurs to me that locked pointers are pretty useless without a `wp_relative_pointer` implementation.

Reply:
Yes, and I've implemented them locally, but I haven't pushed them to github obviously.

Reply:
Maybe relative pointer is an easier protocol that could get in before this one with less effort?

Reply:
I think it would be very stupid to have relative pointer without pointer constraints TBH. Let's just get this done.

Reply:
s/could get in before this one with less effort/could be reviewed in parallel/